etc/docker-test.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
`etc` usually contains configuration files for a *nix system. I understand this is to avoid messing with `bin` which is reserved for the buildpack. How about `script/docker-test`?